code-review/examples/ask.rb

```diff
@@ -0,0 +1,8 @@
+
+class PeopleService
+  def process_people
+    People.connection.execute(
```

Comment:
# Ask if you do not understand something
Is always better if something is complex and, after reading it, you still don't understand, to ask for a description/explanation.

Eg:
`Could you give me a brief explanation about what this query is meant to do? Thanks in advance`